repo/packages/H/hdfs/0/command.json

```diff
@@ -1,5 +1,5 @@
 {
   "pip": [
-    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.0#dcos-hdfs=0.1.0"
+    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.1#dcos-hdfs=0.1.1"
```

Comment:
that fix is already available via 'scheduler-uris' in hdfs package. This is to allow a user to specify individual parameters in options.json.